core/src/com/biglybt/core/internat/MessageText.java

```diff
@@ -866,4 +826,30 @@ public static void revertToDefaultLocale() {
 		Locale	new_locale );
   }
 
+	/**
+	 * @return matching Locale or {@link Locale#ROOT} if unrecognized format or imparsable.
+	 */
+	static Locale parseFormattedLocaleString(String savedLocaleString) {
```

Comment:
What codestyles are accepted? 
I'm using IntelliJ and have imported the codestyle `PreferencesJavaCodeStyleFormatter.xml`

I see other code with different style it makes me anxios; 
```
	public static AllTrackers
	getAllTrackers()
	{
		return( AllTrackersManagerImpl.getSingleton());
	}
```
My IntelliJ likes formatting this into 
```
	public static AllTrackers getAllTrackers() {
		return (AllTrackersManagerImpl.getSingleton());
	}
```
.. but it also displays some warnings:
 * Modifier 'public' is redundant for interface methods
 * Parentheses around '(AllTrackersManagerImpl.getSingleton())' are uneccecary

Resolving the above would result in:
```
	static AllTrackers getAllTrackers() {
		return AllTrackersManagerImpl.getSingleton();
	}
```


I notice that the BiglyBT-Andriod project defines the same formatting.

Reply:
There is no mandated layout - you write code, format it how you want. You edit other people's, try to retain their layout. Don't go reformatting entire files to your preferred layout when making changes.

Reply:
I believe IntelliJ has that option to re-format current block or changed lines only. 
I'll try configuring it so it changes as few lines as possible when refactoring!